src/vm/gcinfodecoder.cpp

```diff
@@ -527,9 +531,22 @@ bool GcInfoDecoder::GetIsVarArg()
     return m_IsVarArg;
 }
 
+#if defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
+bool GcInfoDecoder::HasTailCalls()
+{
+    _ASSERTE( m_Flags & DECODE_HAS_TAILCALLS );
+    return m_HasTailCalls;
+}
+#endif // _TARGET_ARM_ || _TARGET_ARM64_
+
 bool GcInfoDecoder::WantsReportOnlyLeaf()
 {
+    // Only AMD64 with JIT64 can return false here.
```

Comment:
Don't you just want to put the whole function under `#ifdef _TARGET_AMD64_`?

Reply:
I have intentionally kept it on the GCInfoDecoder and just always return true for non-amd64 platforms. The places in the code that call this method would need to have ifdefs around parts of conditions otherwise and it seemed less readable.